src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/__tests__/SettingsEditSettingsTwoFactorAuthTestPage.tsx

```diff
@@ -0,0 +1,85 @@
+import { RootTestPage, expectOne } from "v2/DevTools/RootTestPage"
+
+export class SettingsEditSettingsTwoFactorAuthTestPage extends RootTestPage {
```

Comment:
This is _weird_ — haven't seen this pattern before

Reply:
Was going on the naming convention we have been using so far, which has been the `Settings` prefix + the tab section being migrated (in this instance that is `Edit-Settings`.

Reply:
Not the naming but rather using some class to wrap a bunch of test related methods. Excessive and I'm not sure what RootTestPage is.

Reply:
this is part of the old testing scaffold that David wrote for order app. It shouldn't be used any more, but in this case all this stuff is just code being relocated.